extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCStreamExtractor.java

```diff
@@ -216,7 +219,7 @@ public StreamType getStreamType() {
 
     @Override
     public StreamInfoItemsCollector getRelatedStreams() {
-        return new StreamInfoItemsCollector(getServiceId());
+        return null;
```

Comment:
This was done as per the documentation in the base `getRelatedStreams()`, and also to prevent creating new objects at random. I tested in NewPipe and it works as expected (there are `null` checks), but the code which takes care of it being `null` actually lies [in the extractor](https://github.com/TeamNewPipe/NewPipeExtractor/blob/030465b5d4d8c738a42cfc52bd1d78f54b0f6bd4/extractor/src/main/java/org/schabi/newpipe/extractor/utils/ExtractorHelper.java#L33), where the collector is turned into a list. I now added @Nullable to all of the functions, as it was missing.

Reply:
Oh, and also, YouTube already returns `null` for age restricted videos.